backend/filejump/filejump.go

```diff
@@ -0,0 +1,1680 @@
+		Entries []api.Item `json:"entries"`
+	}
+
+	request := RequestCopy{
+		EntryIds:      []int{srcID},
+		DestinationId: destID,
+	}
+
+	result, err := CallJSONPost[ResultCopy, RequestCopy](f, ctx, "/file-entries/duplicate", &request)
+	if err != nil {
+		return nil, fmt.Errorf("copy failed: %w", err)
+	}
+
+	if result.Status != "success" {
+		return nil, fmt.Errorf("copy failed with status: %s", result.Status)
+	}
+
+	if len(result.Entries) == 0 {
+		return nil, fmt.Errorf("no entries returned from copy operation")
+	}
+
+	// Get the copied entry
+	copiedEntry := &result.Entries[0]
+
+	// If the name doesn't match what we want, rename it
+	if copiedEntry.Name != leaf {
+		// Update the entry name via API
+		type RequestUpdate struct {
+			Name string `json:"name"`
+		}
+
+		updateRequest := RequestUpdate{
+			Name: f.opt.Enc.FromStandardName(leaf),
+		}
+
+		type ResultUpdate struct {
+			Status    string   `json:"status"`
+			FileEntry api.Item `json:"fileEntry"`
+		}
+
+		updateResult, err := CallJSON[ResultUpdate, RequestUpdate](f, ctx, "PUT", "/file-entries/"+copiedEntry.GetID(), nil, &updateRequest)
+		if err != nil {
+			// If rename fails, we still have a successful copy, just with wrong name
+			fs.Debugf(f, "Failed to rename copied file to %q: %v", leaf, err)
+		} else if updateResult.Status == "success" {
+			copiedEntry = &updateResult.FileEntry
+		}
+	}
+
+	// Create new object from the copied entry
+	return f.newObjectWithInfo(ctx, remote, copiedEntry)
+}
+
+// Move src to this remote using server-side move operations.
+//
+// This is stored with the remote path given
+//
+// It returns the destination Object and a possible error
+//
+// Will only be called if src.Fs().Name() == f.Name()
+//
+// If it isn't possible then return fs.ErrorCantMove
+func (f *Fs) Move(ctx context.Context, src fs.Object, remote string) (fs.Object, error) {
+	// For now, use copy + delete as a fallback since the FileJump move API
+	// doesn't seem to work as expected in all cases
+	dstObj, err := f.Copy(ctx, src, remote)
+	if err != nil {
+		return nil, err
+	}
+	
+	// Delete the source object
+	err = src.Remove(ctx)
+	if err != nil {
+		// If delete fails, try to remove the copied object to avoid duplicates
+		_ = dstObj.Remove(ctx)
+		return nil, err
+	}
+	
+	return dstObj, nil
+}
+
+
+// Check the interfaces are satisfied
+var (
+	_ fs.Fs     = (*Fs)(nil)
+	_ fs.Purger = (*Fs)(nil)
+	_ fs.Copier = (*Fs)(nil)
+	_ fs.Mover  = (*Fs)(nil)
+	_ fs.Object = (*Object)(nil)
+)
+
+// type Object interface:
+
+// SetModTime sets the metadata on the object to set the modification date
+func (o *Object) SetModTime(ctx context.Context, t time.Time) error {
+	// fs.Logf(nil, "SetModTime wurde aufgerufen")
+	// FileJump doesn't support setting custom modification times via API
+	// The modification time is set by the server when the file is uploaded
+	// Return this error to indicate that setting mod time requires re-upload
+	return fs.ErrorCantSetModTimeWithoutDelete
```

Comment:
You should just return `fs.ErrorCantSetModTime` here. The `fs.ErrorCantSetModTimeWithoutDelete` is for backends which can only set modification times when they upload a file.